chain/client/src/info.rs

```diff
@@ -239,6 +239,14 @@ impl InfoHelper {
         }
     }
 
+    /// Records protocol version of the current epoch.
+    fn record_protocol_version(head: &Tip, client: &crate::client::Client) {
+        let _ = client
```

Comment:
Code quality idea: a way to avoid `let _` part would be to use pattern matching:

```rust
if let Ok(version) = client.runtime_adapter.get_epoch_protocol_version(&head.epoch_id) {
  metrics::CURRENT_PROTOCOL_VERSION.set(version as i64);
}
```

In my opinion, this is a bit easier to read as it makes it obvious that `get_epoch_protocol_version` returns a single result (as opposed to collection that `map` can operate on) and that we wouldn't log the version if it's not available.